packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -121,12 +121,17 @@ function Label(props: PropsLabel) {
 
   const label = (
     <Tag className={styles.label} htmlFor={inputId}>
-      <Text color="heading" weight="bold">
+      <Text color="heading" fontWeight="bold">
         {children}
       </Text>
       {!required && optionalLabel && (
         <span className={styles.optionalLabel}>
-          <Text color="sub" size="caption" weight="regular" lineHeight="normal">
+          <Text
```

Comment:
Do we need these when the elements already have `className` styles? My thinking is to align with Button above, whose implementation I like.

Reply:
Yes, until this component is wrapped in a Box we do. If we do remove this Text it shouldn't happen until there's a Box wrapper.

Reply:
That makes sense - do we have a Field -> Box ticket already? I can add a note there.